rt_mcca.m

```diff
@@ -134,9 +135,33 @@
     for mm = 1:size(ix,2)
       dataout.trial{ix(m,mm)} = unmixing_flipped(:,:,m,mm) * data.trial{ix(m,mm)};
       dataout.trial{iy(m,mm)} = unmixing_flipped(:,:,m,mm) * data.trial{iy(m,mm)};
+     
+      % facilitates covariance estimate for final polarity check/adjustment
+      dataout.trial{ix(m,mm)} = dataout.trial{ix(m,mm)} - nanmean(dataout.trial{ix(m,mm)},2);
```

Comment:
I am not sure if I understand why the mean needs to be subtracted if ft_timelockanalysis is applied to this data shortly after. Is this the mean over time or repetitions?

Reply:
the mean is subtracted always for the covariance computation, but we want to avoid DC offsets per clip to avoid clip-specific offset to affect the mcca